src/DIRAC/Core/Utilities/JDL.py

```diff
@@ -157,3 +163,161 @@ def dumpCFGAsJDL(cfg, level=1, tab="  "):
+
+        if jdl.lookupAttribute("GridCE"):
+            jobDescription.gridCE = jdl.getAttributeString("GridCE")
+            jdl.deleteAttribute("GridCE")
+
+        if jdl.lookupAttribute("InputData"):
+            jobDescription.inputData = jdl.getListFromExpression("InputData")
+            jdl.deleteAttribute("InputData")
+
+        if jdl.lookupAttribute("InputDataPolicy"):
+            jobDescription.inputDataPolicy = jdl.getAttributeString("InputDataPolicy")
+            jdl.deleteAttribute("InputDataPolicy")
+
+        if jdl.lookupAttribute("InputSandbox"):
+            jobDescription.inputSandbox = jdl.getListFromExpression("InputSandbox")
+            jdl.deleteAttribute("InputSandbox")
+
+        if jdl.lookupAttribute("JobGroup"):
+            jobDescription.jobGroup = jdl.getAttributeString("JobGroup")
+            jdl.deleteAttribute("JobGroup")
+
+        if jdl.lookupAttribute("JobName"):
+            jobDescription.jobName = jdl.getAttributeString("JobName")
+            jdl.deleteAttribute("JobName")
+
+        if jdl.lookupAttribute("JobType"):
+            jobDescription.jobType = jdl.getAttributeString("JobType")
+            jdl.deleteAttribute("JobType")
+
+        if jdl.lookupAttribute("LogLevel"):
+            jobDescription.logLevel = jdl.getAttributeString("LogLevel")
+            jdl.deleteAttribute("LogLevel")
+
+        if jdl.lookupAttribute("NumberOfProcessors"):
+            jobDescription.maxNumberOfProcessors = jdl.getAttributeInt("NumberOfProcessors")
+            jobDescription.minNumberOfProcessors = jdl.getAttributeInt("NumberOfProcessors")
+            jdl.deleteAttribute("NumberOfProcessors")
+            jdl.deleteAttribute("MaxNumberOfProcessors")
+            jdl.deleteAttribute("MinNumberOfProcessors")
+        else:
+            if jdl.lookupAttribute("MaxNumberOfProcessors"):
+                jobDescription.maxNumberOfProcessors = jdl.getAttributeInt("MaxNumberOfProcessors")
+                jdl.deleteAttribute("MaxNumberOfProcessors")
+            if jdl.lookupAttribute("MinNumberOfProcessors"):
+                jobDescription.minNumberOfProcessors = jdl.getAttributeInt("MinNumberOfProcessors")
+                jdl.deleteAttribute("MinNumberOfProcessors")
+
+        if jdl.lookupAttribute("OutputData"):
+            jobDescription.outputData = set(jdl.getListFromExpression("OutputData"))
+            jdl.deleteAttribute("OutputData")
+
+        if jdl.lookupAttribute("OutputSandbox"):
+            jobDescription.outputSandbox = set(jdl.getListFromExpression("OutputSandbox"))
+            jdl.deleteAttribute("OutputSandbox")
+
+        if jdl.lookupAttribute("OutputPath"):
+            jobDescription.outputPath = jdl.getAttributeString("OutputPath")
+            jdl.deleteAttribute("OutputPath")
+
+        if jdl.lookupAttribute("OutputSE"):
+            jobDescription.outputSE = jdl.getAttributeString("OutputSE")
+            jdl.deleteAttribute("OutputSE")
```

Comment:
Just a very minor suggestion that could be applied on every field:

```suggestion
        outputSE = "OutputSE"
        if jdl.lookupAttribute(outputSE):
            jobDescription.outputSE = jdl.getAttributeString(outputSE)
            jdl.deleteAttribute(outputSE)
```

Just to avoid such potential problems:

```python
if jdl.lookupAttribute("OutputSE"):
    jobDescription.outputSE = jdl.getAttributeString("OutputSEs")
    jdl.deleteAttribute("OutputSE")
```